contracts/modules/common/RelayerModule.sol

```diff
@@ -106,9 +89,11 @@ contract RelayerModule is BaseModule {
         require(checkAndUpdateUniqueness(_wallet, _nonce, signHash), "RM: Duplicate request");
         require(verifyData(address(_wallet), _data), "RM: Target of _data != _wallet");
         uint256 requiredSignatures;
-        (requiredSignatures,) = getRequiredSignatures(_wallet, _data);
+        OwnerSignature ownerSignatureRequirement;
+        (requiredSignatures, ownerSignatureRequirement) = getRequiredSignatures(_wallet, _data);
```

Comment:
This can be written as ```(uint256 requiredSignatures, OwnerSignature  ownerSignatureRequirement) = getRequiredSignatures(_wallet, _data);``` which I personally find more readable.